src/Storage/Storage/Blob/Cmdlet/SetAzureStorageBlobContent.cs

```diff
@@ -371,7 +371,7 @@ protected override void DoEndProcessing()
 
         //only support the common blob properties for block blob and page blob
         //http://msdn.microsoft.com/en-us/library/windowsazure/ee691966.aspx
-        private Dictionary<string, Action<StorageBlob.BlobProperties, string>> validCloudBlobProperties =
+        public static Dictionary<string, Action<StorageBlob.BlobProperties, string>> validCloudBlobProperties =
```

Comment:
Could you help me to understand why change from private to public? Would internal be enough? I have this question because we prefer expose PS wrapper object instead of native sdk object.

Reply:
Will see if can move to base class, and make it as protected.